lib/widgets/menu.dart

```diff
@@ -23,7 +29,11 @@ class Menu extends StatelessWidget {
             mainAxisAlignment: MainAxisAlignment.spaceAround,
             crossAxisAlignment: CrossAxisAlignment.start,
             children: <Widget>[
-              Sound(toggleMusic, isPlaying),
```

Comment:
I'm not sure why we are moving from toggleMusic and isPlaying to toggleSound and soundOn